backend/server/controllers/checklists.js

```diff
@@ -243,6 +252,7 @@ module.exports = {
         data: req.body
       })
     } catch (e) {
+      console.error(e)
       logger.error('Checklist creation error.', { error: e.message })
```

Comment:
Doesn't `logger` already log the error?

Reply:
It only shows the error message but lacks other important details, such as line number or other error details